docs/components/base/boat.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Configure a Boat Base"
+linkTitle: "boat"
+weight: 35
+type: "docs"
+draft: "true"
+description: "Configure a boat base."
+tags: ["base", "components"]
+# SMEs: Eliot
+---
+
+A `boat` base is a model for a mobile robotic boat.
+To configure a `boat` base as a component of your robot, first configure the [board](/components/board/) controlling the base and any [motors](/components/motor/) attached to the base.
+
+{{< tabs name="Configure a Boat Base" >}}
+{{% tab name="Config Builder" %}}
+
+On the **COMPONENTS** subtab of your robot's page in [the Viam app](https://app.viam.com), navigate to the **Create Component** menu.
+Enter a name for your base, select the type `base`, and select the `boat` model.
+
+<img src="../img/agilex-limo-ui-config.png" alt="An example configuration for a boat base in the Viam app, with Attributes & Depends On drop-downs and the option to add a frame." style="max-width:900px"/>
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "base",
+      "type": "base",
+      "model": "aboat",
+      "attributes": {
+        "drive_mode": <"a_drive_mode_option">,
+        "serial_path": <"/dev/ttyXXXX">
```

Comment:
It looks like this came from agilex and needs to be changed for boat attributes

Reply:
Boat is draft!

Reply:
Eliot is SME